### PR TITLE
Build simple adapter

### PR DESCRIPTION
**Pull request by Levia**
- Added devise user model
- Added simple country model
- Added adapter model
- Implemented encrypted authentication attributes for adapter
- Implemented base classes for future development of adapters
- Implemented simple adapter taking care of timeouts and downtimes
- Added some tests for simple adapter (can be used as a reference for other adapters) and SOAP transport interface

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -0,0 +1,268 @@
+# Use this hook to configure devise mailer, warden hooks and so forth.
+# Many of these configuration options can be set straight in your model.
+Devise.setup do |config|
+  # The secret key used by Devise. Devise uses this key to generate
+  # random tokens. Changing this key will render invalid all existing
+  # confirmation, reset password and unlock tokens in the database.
+  # Devise will use the `secret_key_base` as its `secret_key`
+  # by default. You can change it below and use your own secret key.
+  config.secret_key = '2eba80cc79049c926bcc0020ef2a31341442d7227d9a9fc8e8dd28a101ef008f7a7ccf5d19d58cde42d06d86d76ea67c52db6482ed627da2df42f34da822f548'
```
Comment (agnessa) [inline config/initializers/devise.rb]: let's pull that from secrets

File: config/initializers/devise.rb [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+
+  # ==> Mailer Configuration
+  # Configure the e-mail address which will be shown in Devise::Mailer,
+  # note that it will be overwritten if you use your own mailer class
+  # with default "from" parameter.
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
```
Comment (agnessa) [inline config/initializers/devise.rb]: let's pull that from secrets

File: test/models/user_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (agnessa) [inline test/models/user_test.rb]: let's remove the empty test and fixture file

**Comment by Levia** [comment]
This is ready to be reviewed again

**Pull request merged**